app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,68 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+class TaskController {
+    private List<Task> tasks = new ArrayList<>();
+    private Long newId = 0L;
+
+    @GetMapping()
+    public List<Task> getTasks() {
+        return tasks;
+    }
+
+    @GetMapping("{taskId}")
+    public ResponseEntity getTask(@PathVariable long taskId) {
+        for(Task storedTask : tasks){
+            if(storedTask.getId() == taskId){
+                return new ResponseEntity(storedTask, HttpStatus.OK);
+            }
+        }
+        return new ResponseEntity(HttpStatus.NOT_FOUND);
+    }
+
+    @PostMapping()
+    public ResponseEntity createTask(@RequestBody Task task) {
+        task.setId(generateId());
+        tasks.add(task);
+
+        return new ResponseEntity(task, HttpStatus.CREATED);
+    }
```

Comment:
상태 코드를 변경하기 위해서 `ResponseEntity`로 반환하고 있는 것 같아요
`@ResponseStatus` 데코레이터를 사용하면 상태코드를 지정할 수 있고, 반환 타입을 `Task`라고 한다면 함수의 Signature만 보더라도 어떤 응답을 줄지 유추할 수 있을거에요
```suggestion
    @PostMapping()
    @ResponseStatus(HttpStatus.CREATED)
    public Task createTask(@RequestBody Task task) {
        task.setId(generateId());
        tasks.add(task);
        return task;
    }
```

Reply:
윤석 트레이너님 감사합니다!